scenario_configs/asr_deepspeech_baseline_fgsm.json

```diff
@@ -17,8 +17,11 @@
         "name": "FastGradientMethod",
         "targeted": true,
         "targeted_labels": {
-            "scheme": "string",
-            "value": "TEST STRING"
+            "args": {
+                "value": "TEST STRING"
+            },
```

Comment:
Two issues here:
1) args here is dictionary, which is confusing. I think we should probably either have this be a list so it can be handled like args, or more helpfully, turned into a kwargs dictionary (which would require modifying the label targeters to take kwargs).
2) The "args" field is never actually used by the `load_label_targeter` function. It doesn't look for config["args"]["value"]. It looks directly for config["value"]

Reply:
I like the kwargs dictionary better than a list of args. R.e. (2), I did confirm that "TEST STRING" was being used. Perhaps this is a moot point after my latest commits